<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1070</width>
    <height>700</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1070</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#393939
</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>2</y>
     <width>531</width>
     <height>690</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>510</width>
     <height>690</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid white;
border-radius:10px;
border:none;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>511</width>
      <height>681</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="img_label">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>400</width>
         <height>600</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>500</width>
         <height>600</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>36</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:white;
border:1px solid white;
padding:10px;</string>
       </property>
       <property name="text">
        <string>Image</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_classify">
       <property name="minimumSize">
        <size>
         <width>400</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Roboto</family>
         <pointsize>22</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:#898987;
color:white;
border-radius:10px;
font-size:18;
font-weight:bold;
</string>
       </property>
       <property name="text">
        <string>Classify image</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>2</y>
     <width>501</width>
     <height>700</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>500</width>
      <height>680</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>500</width>
      <height>680</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1px solid white;
border-radius:10px;</string>
    </property>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>10</y>
       <width>498</width>
       <height>241</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>500</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border-left:none;
border-right:none;
border-radius:0px;
border-color:white;
border-top:none;
</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>481</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:white;
border:none;
text-align:center;</string>
      </property>
      <property name="text">
       <string>image path</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_image_browse">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>450</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">color:white;
background-color:gray;
font-size:30px;
border-radius:10px;</string>
      </property>
      <property name="text">
       <string>browse image file</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txt_image_path">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>481</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:gray;
border:none;
border-radius:10px;
color:white;
padding:2px;
</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_dir_browse">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>180</y>
        <width>450</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">color:white;
background-color:gray;
font-size:30px;
border-radius:10px;</string>
      </property>
      <property name="text">
       <string>browse directory</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="btn_simulate">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>620</y>
       <width>471</width>
       <height>50</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>400</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Roboto</family>
       <pointsize>22</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:#898987;
color:white;
border-radius:10px;
font-size:18;
font-weight:bold;
</string>
     </property>
     <property name="text">
      <string>simualate production line</string>
     </property>
    </widget>
    <widget class="QWidget" name="widget_3" native="true">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>581</y>
       <width>498</width>
       <height>20</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>500</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border-left:none;
border-right:none;
border-radius:0px;
border-color:white;
border-bottom:none;
</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_flag">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>260</y>
       <width>461</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size:30px;
color: rgb(170, 255, 127);
text-align:center;
font-style:bold;
</string>
     </property>
     <property name="text">
      <string>Flag</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_confidence">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>510</y>
       <width>461</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size:30px;
text-align:center;
font-style:bold;
color:rgb(255, 255, 0);

</string>
     </property>
     <property name="text">
      <string>Confidence Rate</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_icon">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>320</y>
       <width>200</width>
       <height>180</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;
border:1px solid white;
border:none;
</string>
     </property>
     <property name="text">
      <string>icon</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
